assets/shuttleSchedule.json

```diff
@@ -1,15 +1,15 @@
 {
   "loyola": {
     "1":[
-      "0730",
-      "0740",
-      "0755",
-      "0820",
-      "0835",
-      "0855",
-      "0910",
-      "0930",
-      "0945",
+      "730",
```

Comment:
wait hold up, why delete all the 0s? we just readded them recently. Can you double check this is as intended.